pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -209,9 +210,12 @@ func (m *manager) RemoveContainer(containerID string) error {
 
 func (m *manager) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAdmitResult {
 	klog.InfoS("Topology Admit Handler")
+	startTime := time.Now()
 
 	metrics.TopologyManagerAdmissionRequestsTotal.Inc()
```

Comment:
Do we want this call inside the measurement for the admission time? I don't feel strongly -- mostly just curious why we would include this but exclude the log?

My intuition would be that the function should look like this (but like I said, I don't feel too strongly):
```
func (m *manager) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAdmitResult {
	klog.InfoS("Topology Admit Handler")
	metrics.TopologyManagerAdmissionRequestsTotal.Inc()
	
	startTime := time.Now()
	podAdmitResult := m.scope.Admit(attrs.Pod)
	metrics.TopologyManagerAdmissionDuration.Observe(float64(time.Since(startTime).Milliseconds()))
	
	return podAdmitResult
}
```

Reply:
I think your suggestion has merit. It makes sense to measure exactly the admission latency and not be impacted by the latency introduced by incrementing the  counter for`TopologyManagerAdmissionRequestsTotal` metric. I will update.